site/content/en/docs/Guides/metrics.md

```diff
@@ -190,8 +190,8 @@ kubectl apply -f ./build/grafana/
 Now we can install [grafana chart](https://github.com/helm/charts/tree/master/stable/grafana) from stable repository. (Replace `<your-admin-password>` with the admin password of your choice)
 
 ```bash
-helm install --wait --name grafana stable/grafana --namespace metrics \
-  --set adminPassword=<your-admin-password> -f ../build/grafana.yaml
+helm install --wait --name grafana stable/grafana --version=5.0.11 --namespace metrics \
```

Comment:
@cyriltovena any issue with specifying a specific version?

Reply:
No you can even make it latest. We release 6.7.3. Always good to be up to date.

Reply:
Which is 5.0.13 btw

Reply:
5.0.13 chart is still appVersion 6.7.1:
https://github.com/helm/charts/blob/866bf2f655ede9aa1bb6f4145db5862035cf49ad/stable/grafana/Chart.yaml#L4

Reply:
5.0.13 is the latest helm chart too. :+1: